iosHackthonApp/Modules/Features/AddMeal/Sources/AddMeal/AddMealViewModel.swift

```diff
@@ -9,30 +9,31 @@
 import shared
 import Components
 import Location
+import Extensions
 
 protocol AddMealViewModelProtocol {
 
-	var code: String { get }
+    var code: String { get }
     var activeAlert: ActiveAlert { get }
-	var showingAlert: Bool { get }
-	var sdk: MealsSDK { get }
-	var locationManager: LocationManager { get }
-	func postMeal(meal: Meal)
+    var showingAlert: Bool { get }
+    var sdk: MealsSDK { get }
+    var locationManager: LocationManager { get }
+    func postMeal(meal: Meal)
 }
 
-final class AddMealViewModel: ObservableObject {
-
-	@Published var code = ""
-	@Published var showingAlert = false
-    @Published var activeAlert: ActiveAlert = .collection
-	
-	let sdk: MealsSDK
-	let locationManager: LocationManager
-	
-	init(sdk: MealsSDK, locationManager: LocationManager) {
-		self.sdk = sdk
-		self.locationManager = locationManager
-	}
+public final class AddMealViewModel: ObservableObject {
+
+    @Published public var code = ""
+    @Published public var showingAlert = false
+    @Published public var activeAlert: ActiveAlert = .collection
+
+    public let sdk: MealsSDK
```

Comment:
Do these need to be public? I know the `init` does, but not so sure about the properties, since I think only the view uses them, which is internal to this module?